SConstruct

```diff
@@ -891,17 +891,16 @@ if env.msvc and not methods.using_clang(env):  # MSVC
         env.AppendUnique(LINKFLAGS=["/WX"])
 
 else:  # GCC, Clang
-    common_warnings = ["-Wenum-conversion"]
```

Comment:
I would prefer to keep this as an empty array so it's declared in all branches.

I know Python works fine with variables defined in each branch but IMO it's a recipe for runtime errors. Also, we may have warnings common to GCC and Clang that we want to enable in the future.

Reply:
I was about to say that it'd be initialized no matter what, but just realized that the errors aren't using an `else` conditional?
```diff
-    elif methods.using_clang(env) or methods.using_emcc(env):
+    else:
```
I assume this is a legacy syntax; will look into cleaning this up in a future SCons pass